<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="radioButton_Homem">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Homem</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Cao">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CÃ£o</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Gato">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gato</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Executar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>112</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Executar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>701</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultado</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Mostrar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>112</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Excluir">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>112</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Gato_Miau">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>160</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Miau</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Cao_AuAu">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Au Au</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Homem_Oi">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Oi</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
